pyfstat/grid_based_searches.py

```diff
@@ -119,7 +118,7 @@ def _get_search_ranges(self):
         else:
             return None
 
-    def inititate_search_object(self):
+    def _initiate_search_object(self):
```

Comment:
I don't think this should ever be called directly by an outside user.

Reply:
Ok. Question: Does it make sense to choose automatically from `ComputeFstat` or `SemiCoherentSearch` depending on the number of segments? I guess the answer is "Obviously yes, why would you specify the number of segments otherwise?" and "Yes, there's a logging line declaring which class is being instantiated"